packages/chips/Chip.js

```diff
@@ -72,53 +87,112 @@ export class Chip extends Component {
     this.props.handleSelect(this.props.id);
   }
 
+  handleRemoveIconClick = (e) => this.foundation_.handleTrailingIconInteraction(e);
+
+  handleTransitionEnd = (e) => this.foundation_.handleTransitionEnd(e);
```

Comment:
This calls the adapter method `notifyRemoval`. If we're aiming to not implement any "notify" adapter methods, we could move `props.handleRemove` to be called directly here instead.